src/index.html

```diff
@@ -7,12 +7,73 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image-container">
+        <img
+          src="./images/imac.jpeg"
+          alt="item card"
+          class="card__image"
+        />
+      </div>
+
+      <div class="card__name">
+        <div class="card__name-main">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
```

Comment:
Instead of using `<br />` to create a new line, consider wrapping the text in separate elements. This improves the readability of your code and makes it easier to style.